testsuite/features/step_definitions/command_steps.rb

```diff
@@ -9,12 +9,23 @@
 
 Then(/^"([^"]*)" should have a FQDN$/) do |host|
   node = get_target(host)
-  result, return_code = node.run('hostname -f')
+  result, return_code = node.run('hostname -f', false)
   result.delete!("\n")
   raise 'cannot determine hostname' unless return_code.zero?
   raise 'hostname is not fully qualified' unless result == node.full_hostname
 end
 
+Then(/^reverse resolution should work for "([^"]*)"$/) do |host|
+  node = get_target(host)
+  result, _rc = node.run('hostname --version', false)
+  if result =~ /^hostname 3\./
```

Comment:
I recommend an `else` on this statement, so we report when this step is actually not checking anything.
That will be the case for clients like sles11sp4 where the hostname version is 1.100

Reply:
yes, the problem is that `hostname` version 1.100 does not provide `-A` option.

will do.

Reply:
done

one of the problems of this step is that it has to work for avahi too, so we cannot use `host`, `nslookup`, or `dig`

Reply:
wait, getent could do the trick

Reply:
changed to use `getent`

Reply:
new version tested locally to work.